### PR TITLE
[14.0][ADD] account_move_vendor_ac_noinv

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

**Comment by adriresu** [comment]
Tested locally, woks fine.

**Review by dalonsod — CHANGES_REQUESTED** [review]
At last, there's a non-covered case: when the purchase and vendor possible options is long enough, a "Search more..." is enabled, then a new wizard could be opened, and this wizard should also be restricted to POs:

imagen


imagen

Could you make the additional changes?

File: account_move_vendor_ac_noinv/views/account_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_move_form_autocomplete_button_purchase_inherit" model="ir.ui.view">
+        <field name="name">account.move.form (Union Autocomplete)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
```
Comment (dalonsod) [inline account_move_vendor_ac_noinv/views/account_move_views.xml]: This inheritance is wrong: this field doesn't exist in this view, in my test database fails indeed. In fact, purchase addon adds it. Could you check?

File: account_move_vendor_ac_noinv/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Account Move Autocomplete Button Filter",
+    "summary": """
+         Change autocomplete button filter to get only purchase orders
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "14.0.1.0.0",
+    'category': "Accounting & Finance",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": ["account"],
```
Comment (dalonsod) [inline account_move_vendor_ac_noinv/__manifest__.py]: Wrong inheritance:
```suggestion
    "depends": ["purchase"],
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Please attend comments

**Comment by dalonsod** [comment]
In last force-push, only minor changes:
* Proper manifest category.
* Reword commit message

**Review by dalonsod — APPROVED** [review]
Let's merge it

**Pull request merged**